### DISCUSSION TITLE
Data Info

### DISCUSSION THREAD

**Discussion by fvapolina** [OpenKBC/multiple_sclerosis_proj#110] [Q&A]
Hello! Can you guys please share a bit more details about the dataset used in this project? The basic things like the size of it, format, type of annotation, classes, how it was obtained? Is it possible for me to get an access to it?

**Comment by swiri021** [comment]
@kicheolkim Can you answer for this question ?

**Comment by kicheolkim** [comment]
Hi @fvapolina 

Thank you for your interest in our dataset. You can find the dataset and related metadata in [the reference paper](https://pubmed.ncbi.nlm.nih.gov/33374005/) and in [GEO](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137143).
